recipes-nemomobile/mlite/mlite/0002-mlite-Disable-tests.patch

```diff
@@ -0,0 +1,24 @@
+From e0ea6c4315ca595c08ecff16169f9bb1fb081bd4 Mon Sep 17 00:00:00 2001
+From: MagneFire <IDaNLContact@gmail.com>
+Date: Sat, 16 Oct 2021 19:33:26 +0200
+Subject: [PATCH 1/2] mlite: Disable tests.
+
+---
+ mlite.pro | 1 -
+ 1 file changed, 1 deletion(-)
+
+diff --git a/mlite.pro b/mlite.pro
+index 48bed35..6be58c5 100644
+--- a/mlite.pro
++++ b/mlite.pro
+@@ -1,7 +1,6 @@
+ TEMPLATE = subdirs
+ SUBDIRS = \
+         src \
+-        tests \
+         tools
+ 
```

Comment:
Can you send me a full log of this? I do not get any warning or error.

My full log is as follows:
```
DEBUG: Executing python function extend_recipe_sysroot
NOTE: Direct dependencies are ['virtual:native:/home/darrel/AsteroidOS/asteroid-honisterv2/src/oe-core/meta/recipes-devtools/patch/patch_2.7.6.bb:do_populate_sysroot', '/home/darrel/AsteroidOS/asteroid-honisterv2/src/oe-core/meta/recipes-devtools/quilt/quilt-native_0.66.bb:do_populate_sysroot']
NOTE: Installed into sysroot: []
NOTE: Skipping as already exists in sysroot: ['patch-native', 'quilt-native', 'libtool-native', 'automake-native', 'attr-native', 'autoconf-native', 'texinfo-dummy-native', 'gettext-minimal-native', 'gnu-config-native', 'm4-native']
DEBUG: Python function extend_recipe_sysroot finished
DEBUG: Executing python function do_patch
DEBUG: Executing python function patch_do_patch
DEBUG: Searching for 0001-MDesktopEntry-Allow-dynamic-locale-switch-by-disabli.patch in paths:
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite-+gitAUTOINC+37ded5a0ca/asteroid
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite/asteroid
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/files/asteroid
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite-+gitAUTOINC+37ded5a0ca/sturgeon
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite/sturgeon
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/files/sturgeon
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite-+gitAUTOINC+37ded5a0ca/armv7ve
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite/armv7ve
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/files/armv7ve
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite-+gitAUTOINC+37ded5a0ca/arm
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite/arm
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/files/arm
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite-+gitAUTOINC+37ded5a0ca/
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite/
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/files/
DEBUG: Searching for 0002-mlite-Disable-tests.patch in paths:
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite-+gitAUTOINC+37ded5a0ca/asteroid
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite/asteroid
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/files/asteroid
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite-+gitAUTOINC+37ded5a0ca/sturgeon
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite/sturgeon
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/files/sturgeon
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite-+gitAUTOINC+37ded5a0ca/armv7ve
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite/armv7ve
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/files/armv7ve
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite-+gitAUTOINC+37ded5a0ca/arm
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite/arm
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/files/arm
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite-+gitAUTOINC+37ded5a0ca/
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/mlite/
    /home/darrel/AsteroidOS/asteroid-honisterv2/src/meta-asteroid/recipes-nemomobile/mlite/files/
NOTE: Applying patch '0001-MDesktopEntry-Allow-dynamic-locale-switch-by-disabli.patch' (../src/meta-asteroid/recipes-nemomobile/mlite/mlite/0001-MDesktopEntry-Allow-dynamic-locale-switch-by-disabli.patch)
NOTE: Applying patch '0002-mlite-Disable-tests.patch' (../src/meta-asteroid/recipes-nemomobile/mlite/mlite/0002-mlite-Disable-tests.patch)
DEBUG: Python function patch_do_patch finished
DEBUG: Python function do_patch finished
DEBUG: Executing python function do_qa_patch
DEBUG: Python function do_qa_patch finished

```

Reply:
Okay, had a further look into this, It seems that the generated patch files by `git` (`git format-patch`) always have trailing whitespaces if there was an empty line.

Do you have your git settings setup to be very strict when it comes to white space? Because generally you can just ignore these errors.

I don't think it is possible to solve all whitespace issues as the input source code already has the whitespace issue to begin with (example: https://github.com/AsteroidOS/meta-asteroid/blob/master/recipes-core/psplash/files/0001-Don-t-draw-progress-and-message-bar.patch)

For reference, I have applied the command listed https://wiki.gnome.org/Git/WorkingWithPatches: `emacs <filename> -f delete-trailing-whitespace -f save-buffer -f kill-emacs` to all patch files using:
```
find . -name "*.patch" -type f -exec emacs {} -f delete-trailing-whitespace -f save-buffer -f kill-emacs \;
```
This removes trailing whitespace for all patch files (135 files) but results in compilation issues due to the input code already containing whitespace (`libmlocale` and `psplash` are currently the first few that were encountered).

Reply:
As of the latest commit, this should be fixed (`libmlocale` and `psplash` are the only remaining ones with trailing whitespace).

Let me know if this fixes it for you.